engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -36,14 +36,29 @@ <h4 translate>Basic Information</h4>
     <h4 translate>Sync Settings</h4>
     <dl class="dl-horizontal dl-horizontal-left">
       <span ng-show="repository.content_type === 'yum'" >
-        <dt translate>Restrict to architecture</dt>
+        <dt>
+          <span translate>Restrict to architecture</span>
+          <i class="pficon-info" title="{{ 'The repository will be enabled by default on content hosts with the selected architecture. Select \'Default\' to enable regardless of architecture.' | translate }}"></i>
```

Comment:
What do you think about making the current 'Default' option 'No restriction' instead? IMO that is more clear than Default, and removes the need to explain that in the tooltip (Select Default ...) Likewise for OS versions

Reply:
I do like 'No restriction' as a name.  But I do not think rjerrido will appreciate his tooltip going away completely.

Reply:
what if we change it to 'No Restriction' and keep the tooltips, and just delete the 'Select Default...' sentence?  Is that what you were thinking?

Reply:
Just what I was thinking!